lib/widgets/footer.dart

```diff
@@ -158,18 +104,71 @@ class Footer extends StatelessWidget {
       ),
     );
   }
+}
+
+class SocialMediaButton extends StatelessWidget {
+  const SocialMediaButton({super.key, required this.imagePath, this.targetUrl});
 
-  Widget getSocialMediaButton(imagePath, {String? targetUrl}) {
+  final String imagePath;
+  final String? targetUrl;
+
+  @override
+  Widget build(BuildContext context) {
     return SizedBox(
       height: 48.0,
       width: 48.0,
       child: IconButton(
         padding: const EdgeInsets.all(0.0),
         icon: Image.asset(imagePath),
         onPressed: () async {
-          if (targetUrl != null) openUrl(targetUrl);
+          if (targetUrl != null) Footer.openUrl(targetUrl!);
```

Comment:
calling `Footer.openUrl()` seems little weird.
What about defining `openUrl()` as a global function in footer.dart?